raaghu-components/src/rds-comp-payment-tenure/rds-comp-payment-tenure.tsx

```
@@ -105,16 +106,16 @@ const RdsCompPaymentTenure = (props: RdsCompPaymentProps) => {
             ))}
           <div className="pt-3">
             <div className="mb-4">
-              <p className="fs-6 text-secondary-para mb-0">3 developers already included, you can add additional users here</p>
+              <p className="fs-6 text-secondary-para mb-3">3 developers already included, you can add additional users here</p>
               <RdsCounter
                 key={developerCountValue} 
-                colorVariant="outline-primary"
+                colorVariant="primary"
```

ESLint

[error] Expected indentation of 32 spaces but found 16.

(indent)